### PR TITLE
Notebooks for Diffuser Community Scripts-3

### PR DESCRIPTION
**Pull request by ParagEkbote**
# What does this PR do?

I've created 4 New Notebooks for Diffuser Community Scripts, same as #527. After this PR gets merged, I will update the Colab column in the ReadMe.

Please let me know if any improvements are needed and I will make the necessary changes.

Reference: https://github.com/huggingface/diffusers/blob/main/examples/community/README.md

## Who can review?

 @stevhliu

### CODE REVIEW

File: diffusers/long_prompt_weighting_stable_diffusion.ipynb [hunk 1]
```diff
@@ -0,0 +1,387 @@
+{
```
Comment (stevhliu) [inline diffusers/long_prompt_weighting_stable_diffusion.ipynb:1]: <p>The onnxruntime code snippet is missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: diffusers/stable_diffusion_interpolation.ipynb [hunk 2]
```diff
@@ -0,0 +1,306 @@
+{
```
Comment (stevhliu) [inline diffusers/stable_diffusion_interpolation.ipynb:1]: <p>Missing context about hardware requirements and the output of the <code>walk</code> function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by stevhliu — COMMENTED** [review]
Cool thanks!

**Comment by ParagEkbote** [comment]
I have updated the mentioned notebooks as per the advice given. Could you please review the changes?

cc: @stevhliu

**Review by stevhliu — APPROVED** [review]
LGTM 👍

**Pull request merged**